pkg/cover/html.go

```diff
@@ -516,8 +546,23 @@ func fixUpPCs(target string, progs []Prog, coverFilter map[uint32]uint32) []Prog
 	return progs
 }
 
-func fileContents(file *file, lines [][]byte, haveProgs bool) string {
+type lineCoverExport struct {
+	Module    string
+	Filename  string
+	Covered   []int
+	Uncovered []int
+	Both      []int
+}
+
+func fileContents(file *file, lines [][]byte, haveProgs bool) (string, lineCoverExport) {
 	var buf bytes.Buffer
+	lce := lineCoverExport{
+		Module:    file.module,
+		Filename:  file.filename,
+		Covered:   make([]int, 0),
```

Comment:
Remove these `make([]int, 0)` lines, zero value for a slice is an empty slice.

Reply:
Done